contracts/legacy-contracts/smart-wallet/interface/IAccountPermissions.sol

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+/// @author thirdweb
+
+interface IAccountPermissions {
```

Comment:
should be named differently maybe a `IAccountPermission_V1` otherwise it will clash with the non legacy interface when imported in the SDK